ee/codegen/src/generators/nodes/templating-node.ts

```diff
@@ -46,12 +46,9 @@ export class TemplatingNode extends BaseSingleFileNode<
       python.field({
         name: "inputs",
         initializer: python.TypeInstantiation.dict(
-          otherInputs.map((input) => ({
-            key: python.TypeInstantiation.str(input.key),
-            value: new NodeInputValuePointer({
-              workflowContext: this.workflowContext,
-              nodeInputValuePointerData: input.value,
-            }),
+          otherInputs.map((codeInput) => ({
```

Comment:
switch this to how the other nodes do this to have a single consumer of `NodeInputValuePointer`